financial-templates-lib/test/price-feed/BalancerPriceFeed.js

```diff
@@ -0,0 +1,71 @@
+const winston = require("winston");
+const lodash = require("lodash");
+
+const { BalancerPriceFeed } = require("../../price-feed/BalancerPriceFeed");
+const {
+  mineTransactionsAtTime,
+  advanceBlockAndSetTime,
+  MAX_SAFE_JS_INT,
+  stopMining,
+  startMining
+} = require("@umaprotocol/common");
+const { delay } = require("../../helpers/delay.js");
+
+const BalancerMock = artifacts.require("BalancerMock");
+const Balancer = artifacts.require("Balancer");
+
+contract("balancer price feed", async function(accounts) {
+  describe("BalancerPriceFeed.js", function() {
```

Comment:
nit: why do you have a contract and a describe? I think you could just remove the describe and it would work identically, right?

Reply:
i think this a remnant of combining the tests at one point. will update